### ISSUE TITLE
Lack of warmup in calc test

### ISSUE THREAD

**Issue by terminalsin** [huzpsb/JavaObfuscatorTest#7]
**Issue**
Calc test is consistent due to a lack of warmup

**Solution**
Add a call for each method _before_ it is iterated. Then, iterate the methods as done previously. Then repeat the class 10x times, and only then take their mean and median and measure those. This would be a much more accurate way at measuring performance respective to the obfuscator.

**Comment by huzpsb** [comment]
Hmm...
[s]Can I say this check is made alike this simply because I want to count the warm-up time in? With things like MyJ2C+VMP, the warm up time can be up to 200ms per class and I found that annoying.[/s]
Maybe I would add a seperate flag for warm-up. Thank you for your suggestion.

**Comment by terminalsin** [comment]
> Hmm... [s]Can I say this check is made alike this simply because I want to count the warm-up time in? With things like MyJ2C+VMP, the warm up time can be up to 200ms per class and I found that annoying.[/s] Maybe I would add a seperate flag for warm-up and modify this check accoardingly. Thank you for your suggestion.

Yeah but once executed and cached, JIT assists and makes execution flawless. Performance should measuring consistent calls in an application, not one time loading schemes.
